python/cudf/cudf/core/column/string.py

```diff
@@ -3616,6 +3619,55 @@ def normalize_spaces(self, **kwargs):
             cpp_normalize_spaces(self._column), **kwargs
         )
 
+    def normalize_characters(self, do_lower=True, **kwargs):
+        """
+        Normalizes strings characters for tokenizing.
+
+        This uses the normalizer that is built into the
+        subword_tokenize function which includes:
+
+            - adding padding around punctuation (unicode category starts with
+              "P") as well as certain ASCII symbols like "^" and "$"
+            - adding padding around the CJK Unicode block characters
+            - changing whitespace (e.g. `"\t", "\n", "\r"`) to space `" "`
```

Comment:
@galipremsagar do we need to escape `\` characters for Sphinx?

Reply:
Looks like I need to add another `\` to these. I see this in other similar places in the file. Example:
https://github.com/rapidsai/cudf/blob/79bd43517407302f4b2e52b5b470fdfcafe51a6b/python/cudf/cudf/core/column/string.py#L963

Reply:
Yep, we will need `\` for this kind of instance too inside backticks, else would result in broken docs.

Reply:
Would recommend this: `(e.g. ``\\t``, ``\\n``, ``\\r``)` as having single back-tick seems to be not representing correctly in the end output for this text.